src/app/pages/edit-contact/editContact.component.html

```diff
@@ -0,0 +1,82 @@
+<div class="title">
+  <h1>Edit Contact Page</h1>
+</div>
+
+<div class="edit-contact-container">
+  <form>
+
+    <div>
+      <label for="name">Name: </label>
+    </div>
+    <div>
+      <input class="input" type="text" name="name" [(ngModel)]="this.editContactData.name">
+    </div>
+
+    <div>
+      <label for="name">Address: </label>
+    </div>
+    <div>
+      <input class="input" type="text" name="address" [(ngModel)]="this.editContactData.address">
+    </div>
+
+    <div>
+      <label for="name">Mobile: </label>
```

Comment:
You probably want the same validation you had on your add page on this one as well for all the phone numbers and the email input.